docs/stoploss.md

```diff
@@ -11,32 +11,55 @@ Most of the strategy files already include the optimal `stoploss` value.
 
 ## Stop Loss On-Exchange/Freqtrade
 
-Those stoploss modes can be *on exchange* or *off exchange*. If the stoploss is *on exchange* it means a stoploss limit order is placed on the exchange immediately after buy order happens successfully. This will protect you against sudden crashes in market as the order will be in the queue immediately and if market goes down then the order has more chance of being fulfilled.
+Those stoploss modes can be *on exchange* or *off exchange*.
 
-In case of stoploss on exchange there is another parameter called `stoploss_on_exchange_interval`. This configures the interval in seconds at which the bot will check the stoploss and update it if necessary.
+These modes can be configured with these values:
+
+``` python
+    'emergencysell': 'market',
+    'stoploss_on_exchange': False
+    'stoploss_on_exchange_interval': 60,
+    'stoploss_on_exchange_limit_ratio': 0.99
+```
+
+### stoploss_on_exchange
+Enable or Disable stop loss on exchange.
+If the stoploss is *on exchange* it means a stoploss limit order is placed on the exchange immediately after buy order happens successfully. This will protect you against sudden crashes in market as the order will be in the queue immediately and if market goes down then the order has more chance of being fulfilled.
+
+If `stoploss_on_exchange` uses limit orders, the exchange needs 2 prices, the stoploss_price and the Limit price.
+`stoploss` defines the stop-price - and limit should be slightly below this.  
+If an exchange supports both limit and market stoploss orders, then the value of `stoploss` will be used to determine the stoploss type.
```

Comment:
I would add the `stoploss_on_exchange_limit_ratio` explanation from below in this area ...
Here, we mention that we may need 2 prices, but not how to define the 2nd price ...

Reply:
please take a look at the text now